tests/unit/amo/components/TestCategory.js

```diff
@@ -1,14 +1,16 @@
-import { shallow } from 'enzyme';
+import { shallow, mount } from 'enzyme';
 import React from 'react';
+import { Provider } from 'react-redux';
 
 import { CategoryBase, mapStateToProps } from 'amo/components/Category';
 import { SearchBase } from 'amo/components/Search';
 import createStore from 'amo/store';
 import { categoriesFetch } from 'core/actions/categories';
 import { searchStart } from 'core/actions/search';
 import { ADDON_TYPE_THEME } from 'core/constants';
+import I18nProvider from 'core/i18n/Provider';
 import { getFakeI18nInst } from 'tests/unit/helpers';
-
+import NotFound from 'amo/components/ErrorPage/NotFound';
```

Comment:
Nitpick but this should be above Line 5 `import { SearchBase } from 'amo/components/Search'; `. Not sure why ESLint didn't catch it 😢